### PR TITLE
Implement ajax_add action

### PR DESCRIPTION
**Pull request by yakky**
Allows to define a toolbar action that adds a plugin
Adds the ability to define custom per-toolbar item actions in the process

### CODE REVIEW

**Comment by yakky** [comment]
@mkoistinen @ojii @FinalAngel  ready  for review

**Comment by mkoistinen** [comment]
lgtm

File: cms/static/cms/js/modules/cms.plugins.js [hunk 1]
```diff
@@ -456,6 +456,16 @@ $(document).ready(function () {
 			}
 		},
 
+		editPluginPostAjax: function(caller, toolbar, response){
+			if (typeof toolbar == 'undefined' || typeof response == 'undefined') {
+				return function(toolbar, response) {
+					var that = caller;
+					that.editPlugin(response['url'], that.options.plugin_name, response['breadcrumb']);
```
Comment (FinalAngel) [inline cms/static/cms/js/modules/cms.plugins.js]: just use response.url

File: cms/static/cms/js/modules/cms.plugins.js [hunk 2]
```diff
@@ -456,6 +456,16 @@ $(document).ready(function () {
...
+				}
+			}
+			that.editPlugin(response['url'], that.options.plugin_name, response['breadcrumb']);
```
Comment (FinalAngel) [inline cms/static/cms/js/modules/cms.plugins.js]: same here

**Comment by FinalAngel** [comment]
added some comments, even if too late :)

**Comment by yakky** [comment]
@FinalAngel it's never too late :)

**Pull request merged**